### PR TITLE
V22F-639 Add position to recorder

### PR DESCRIPTION
**Pull request by suppergerrie2**

### CODE REVIEW

File: speed-dreams/source-2.2.3/src/drivers/replaydriver/Driver.cpp [hunk 1]
```diff
@@ -25,16 +25,18 @@ Driver::Driver(int p_index, const char* p_name)
 /// @param p_situation The current race situation
 void Driver::InitTrack(tTrack* p_track, void* p_carHandle, void** p_carParmHandle, tSituation* p_situation)
 {
-    std::experimental::filesystem::path sdaFolder;
+    using std::experimental::filesystem::path;
```
Comment (KungFuDonkey) [inline speed-dreams/source-2.2.3/src/drivers/replaydriver/Driver.cpp:28]: this is fine, but maybe consider shortening the namespace E.G. 
```
namespace filesystem = std::experimental::filesystem
```

Comment (suppergerrie2) [reply speed-dreams/source-2.2.3/src/drivers/replaydriver/Driver.cpp:28]: Will that cause issues if we ever update our c++ version?

Comment (KungFuDonkey) [reply speed-dreams/source-2.2.3/src/drivers/replaydriver/Driver.cpp:28]: Well yes but then its easier to fix as we can just change std::expreimental::filesystem to std::filesystem

**Review by KungFuDonkey — APPROVED** [review]

File: speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp [hunk 2]
```diff
@@ -37,31 +51,31 @@ Recorder::Recorder(const std::string& p_dirName,
...
+    CreateRecordingFile(m_recordingDir, m_userInputRecordingFile, USER_INPUT_RECORDING_FILE_NAME);
+    CreateRecordingFile(m_recordingDir, m_decisionsRecordingFile, DECISIONS_RECORDING_FILE_NAME);
+    CreateRecordingFile(m_recordingDir, m_simulationDataRecordingFile, SIMULATION_DATA_RECORDING_FILE_NAME);
 
     // initialize previous input with impossible values. This ensures the first actual values are always written when compression is enabled.
     m_prevUserInput = new float[m_userParamAmount];
     for (int i = 0; i < m_userParamAmount; i++)
     {
         m_prevUserInput[i] = 2.0f;  // 2.0f is impossible user input
     }
```
Comment (noutheesink) [inline speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Recorder.cpp:63]: Might be better to use NAN here as well
```suggestion
    {
        m_prevUserInput[i] = NAN;
    }
```

**Review by noutheesink — CHANGES_REQUESTED** [review]

**Review by noutheesink — APPROVED** [review]

**Pull request merged**